apps/modeling-app/src/hooks/useAvailableModels.ts

```diff
@@ -0,0 +1,19 @@
+import { ApiError, DefaultService, ModelSpec } from "@dhis2-chap/ui";
+import { useQuery } from "@tanstack/react-query";
+
+export const useAvailableModels = () => {
+    const { data, error, isLoading } = useQuery<ModelSpec[], ApiError>({
+        queryKey: ['availableModels'],
+        queryFn: () => DefaultService.listModelsListModelsGet(),
+        staleTime: Infinity,
+        cacheTime: Infinity,
+        retry: 0,
+    });
```

Comment:
@cursoragent These are actually model templates. Let's rename the hook and querykey accordingly